includes/class-stats-dashboard.php

```diff
@@ -212,6 +212,12 @@ private function get_daily_stats_chart( \WP_Post $job ): array {
 	private function get_stat_summaries( \WP_Post $job ) {
 		$job_stats = new Job_Listing_Stats( $job->ID );
 
+		$views              = $job_stats->get_event_total( Job_Listing_Stats::VIEW );
+		$views_unique       = $job_stats->get_event_total( Job_Listing_Stats::VIEW_UNIQUE );
+		$views_repeat       = $views - $views_unique;
+		$search_impressions = $job_stats->get_event_total( Job_Listing_Stats::SEARCH_IMPRESSION );
+		$search_clicks      = $views_unique / $search_impressions * 100;
```

Comment:
If `$search_impressions` here is `0` then this could be a fatal `Division by zero`.  Maybe we should add a defensive check here to only do search clicks if there are impressions?

Reply:
Added a check in 15e705b